libs/log-filters.js

```diff
@@ -1,6 +1,6 @@
 const levels = {
-  debug: 0,
   secret: 0,
+  debug: 10,
```

Comment:
shouldn't other levels be increased by 10 ?

Reply:
Right